pkg/service/endpoints_controller.go

```diff
@@ -304,7 +305,20 @@ func (e *EndpointController) syncService(key string) {
 
 			portName := servicePort.Name
 			portProto := servicePort.Protocol
+
 			portNum, err := findPort(pod, servicePort)
+
+			// handle DEPRECATED undeclared numeric ports gracefully for now.
```

Comment:
does our mesos endpoint controller do this already (log the warning?) or fail? I can't find a warning in that code so I'm assuming that it fails.